ledger-core-ethereum/inc/ethereum/operations/EthereumLikeOperationQuery.hpp

```diff
@@ -28,6 +28,8 @@
  *
  */
 
+#pragma once
```

Comment:
That one was tricky to find in the first place! :)

Reply:
Meh. Dunno what happen in my life during that segregation.. But obviously bad things ^^

Reply:
![](https://phaazon.net/media/uploads/ohno.jpg)